spec/acceptance/api/method_spec.rb

```diff
@@ -44,7 +44,7 @@
 
   json(:resource) do
     let(:root) { 'method' }
-    it { should have_properties('id', 'name', 'system_name', 'friendly_name') }
+    it { should have_properties('id', 'name', 'system_name', 'friendly_name', 'description') }
```

Comment:
This is failing because the metric/method built before contains `description` as `nil` and that's why it is not returned. So you can change this line:
https://github.com/3scale/porta/blob/05adee12939a479951fb6f54ae67fceb5c3077bc/spec/acceptance/api/method_spec.rb#L11
For something like:
```ruby
let(:resource) { FactoryBot.build(:metric, service: service, parent: hits, description: 'metric description') }
```
And it works then 😄